make-release.sh

```diff
@@ -395,13 +395,16 @@ bumpVersion() {
 }
 
 updateImageTagsInCheServer() {
-    cd che-server
-    git checkout ${BRANCH}
-    cd .ci
-    ./set_tag_version_images.sh ${CHE_VERSION}
-    cd ..
-    git commit -asm "chore: Set ${CHE_VERSION} release image tags"
-    git push origin ${BRANCH}
+    pushd che-server >/dev/null
+        git checkout ${BRANCH}
+        pushd .ci >/dev/null
+            ./set_tag_version_images.sh
```

Comment:
this doesn't need to have `${CHE_VERSION}` passed in?

Reply:
not anymore, the script used to update many entries in che.properties, and probably helm charts, but now it's reduced to two entries which just use the constant "latest" string (and thus, I remove the remains from the script into make-release.sh)